packages/grant-explorer/src/features/round/ViewCartPage/RoundInCart.tsx

```diff
@@ -15,14 +23,84 @@ export function RoundInCart(
     String(props.roundCart[0].chainId),
     props.roundCart[0].roundId
   ).round;
+
   const minDonationThresholdAmount =
     round?.roundMetadata?.quadraticFundingConfig?.minDonationThresholdAmount ??
     1;
+
+  const { address } = useAccount();
+  const votingTokenForChain = useCartStorage((state) =>
+    state.getVotingTokenForChain(props.roundCart[0].chainId)
```

Comment:
Yes, the RoundInCart component doesn't get rendered unless there are projects in this round. But a valid concern nevertheless, and I added a guard just in case. the getVotingTokenForChain handles an undefined arg gracefully.